src/store/metadata/stub/content/editions/versions/cancer-diagnosis-at-stage-1-and-2_2023-05.json

```diff
@@ -0,0 +1,76 @@
+{
+    "@context": "https://staging.idpd.uk/ns#",
+    "@id": "https://staging.idpd.uk/datasets/cancer-diagnosis-at-stage-1-and-2/editions/2023-05/versions",
+    "@type": "hydra:Collection",
+    "title": "Cancer diagnosis at stage 1 and 2",
+    "versions": [
+        {
+            "@type": [
+                "dcat:Distribution",
+                "csvw:Table"
+            ],
+            "@id": "https://staging.idpd.uk/datasets/cancer-diagnosis-at-stage-1-and-2/editions/2023-05/versions/1",
+            "identifier": "1",
+            "issued": "2023-05-17T12:25:53Z",
+            "title": "Cancer diagnosis at stage 1 and 2",
+            "summary": "New cases of cancer diagnosed at stages 1 and 2 as a percentage of all new cases of cancer diagnosed at any known stage, England, 2013 to 2020.",
+            "description": "This dataset shows new cases of cancer diagnosed at stages 1 and 2 as a percentage of all new cases of cancer diagnosed at any known stage, in England, for 2013 to 2020.",
+            "download_url": "https://staging.idpd.uk/datasets/cancer-diagnosis-at-stage-1-and-2/editions/2023-05/versions/1.csv",
+            "media_type": "text/csv",
+            "table_schema": {
+              "columns": [
+                {
+                  "name": "period",
+                  "datatype": "string",
+                  "titles": "Year",
+                  "description": "The time period associated with the observation."
+                },
+                {
+                  "name": "areacd",
+                  "datatype": "string",
+                  "titles": "Statistical Geography",
+                  "description": "The statistical geography associated with the observation."
+                },
+                {
+                  "name": "value",
```

Comment:
Just for consistency between other json metadata files change this to 'observation'.